doc/10-Using-ImageJ-Ops.ipynb

```diff
@@ -125,9 +126,16 @@
    "execution_count": 4,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "The compare_version function is deprecated. Use is_version_at_least instead.\n"
```

Comment:
Let's fix these usages of deprecated functions, so they don't clutter up the notebooks. I pushed 47c36dd3fc329ee85d5981b7676d5e70e0ba1e76 which stops using `compare_version`. You can rebase this PR on top of the latest master branch and regenerate these outputs to eliminate this cruft.

Reply:
Should be clean now!